packages/features/ee/workflows/components/AddActionDialog.tsx

```diff
@@ -1,8 +1,3 @@
-/**
```

Comment:
All diffs in this page are because the v1 version had the same name, and I moved v2 to the same place. So this is actually the difference between v1 and v2.